### PR TITLE
Sandbox responsivenesss

### PR DESCRIPTION
**Pull request by langolf**
=

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -15,4 +15,12 @@ module.exports = {
             },
         ],
     ],
+    plugins: [
+        [
+            require.resolve('babel-plugin-styled-components'),
```
Comment (Winner95) [inline babel.config.js:20]: what is the purpose of this change?

It seems like display name is true by default in this plugin.

It is a good opportunity to fix our tests for each PR - please make a separate PR and make adjustment for Jest with https://styled-components.com/docs/tooling#usage - Server-side rendering

File: get-webpack-config.js [hunk 2]
```diff
@@ -136,7 +136,7 @@ module.exports = function getWebpackConfig({
             clientLogLevel: isDebug ? 'info' : 'warning',
             contentBase: path.resolve(__dirname, 'dist'),
             hot: true,
-            open: true,
+            open: !isDebug,
```
Comment (Winner95) [inline get-webpack-config.js:139]: I don't recommend to make this change.

currently isDebug is only used for debugging console output - please revert it. 

If you would like this change - please send separate PR, so we can discuss it.

File: package.json [hunk 3]
```diff
@@ -76,6 +76,7 @@
   },
   "devDependencies": {
     "@types/react": "16.8.4",
+    "babel-plugin-styled-components": "^1.12.0",
```
Comment (Winner95) [inline package.json:79]: Currently we prefer strict versions in this project for dependencies, which affects bundle

File: src/components/app-view/app-view.jsx [hunk 4]
```diff
@@ -22,17 +22,25 @@ const AppSidebar = styled(Sidebar)`
...
-    width: calc(100% - ${SIDEBAR_WIDTH}px);
-    transition: transform 0.2s cubic-bezier(0.165, 0.84, 0.44, 1);
-    will-change: transform;
+    right: 0;
+    left: 0;
+    height: 100vh;
+    transition: left 0.3s cubic-bezier(0.165, 0.84, 0.44, 1);
+    will-change: left;
+
+    @media screen and (min-width: ${deviceSizes.tablet}px) {
```
Comment (Winner95) [inline src/components/app-view/app-view.jsx:35]: can we add check for for deviceSizes.tablet that it is number and if so - add px? we can make a separate helper for it.

File: src/components/app-view/app-view.jsx [hunk 5]
```diff
@@ -138,7 +137,7 @@ class AppView extends React.Component {
                     navigation={this.props.navigation}
                 ></AppSidebar>
 
-                <Content>{this.props.content}</Content>
+                <Content isSidebarVisible={this.state.sidebarOpened}>{this.props.content}</Content>
```
Comment (Winner95) [inline src/components/app-view/app-view.jsx:140]: a small note to change, which didn't occur to me last time - `isSidebarVisible` and `sidebarOpened ` - can we have a consistent naming for both, please?

**Review by Winner95 — CHANGES_REQUESTED** [review]

**Comment by Winner95** [comment]
@langolf  thank you for your contribution! this PR looks good in 85%, please read my notes and fix conflicts with current main branch.

File: src/visual-helpers/styleguide-cell/styleguide-cell.jsx [hunk 6]
```diff
@@ -35,22 +50,22 @@ function StyleguideCell(props) {
...
-                    height,
                     borderWidth: border ? 1 : 0,
                     borderColor: '#aaa',
                     borderStyle: 'solid',
-                    overflow: 'auto',
                     transform:
                         Platform.OS === 'web' ? 'translate3d(0, 0, 0)' : [{ translateZ: '0' }],
                 }}
             >
+                <svg viewBox={`0 0 ${width} ${height}`} />
```
[the remark this answers is not in the document]
Comment (langolf) [reply src/visual-helpers/styleguide-cell/styleguide-cell.jsx:65]: This is one of aspect-ratio technique to keep frames aspect and at the same time give them responsiveness.

File: src/components/app-view/app-view.jsx (hunk 4 above)
Comment (langolf) [reply src/components/app-view/app-view.jsx:35]: Done. It will be more like a sass-mixin then utility. It depends on "styled-components" so I define it inside app styles.

**Comment by langolf** [comment]
I've fixed all issues. It's time to fix master :)

File: get-webpack-config.js (hunk 2 above)
Comment (langolf) [reply get-webpack-config.js]: fixed

File: src/components/app-view/app-view.jsx (hunk 4 above)
Comment (langolf) [reply src/components/app-view/app-view.jsx]: Check last update. Should I use it over al project?

**Comment by Winner95** [comment]
@langolf - thank you! please merge master in your branch)
